src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -23,11 +24,30 @@ public class HttpSysOptions
         private RequestQueue _requestQueue;
         private UrlGroup _urlGroup;
         private long? _maxRequestBodySize = DefaultMaxRequestBodySize;
+        private string _requestQueueName;
 
         public HttpSysOptions()
         {
         }
 
+        /// <summary>
+        /// The name of the Http.Sys request queue
+        /// </summary>
+        public string RequestQueueName
+        {
+            get => _requestQueueName;
+            set
+            {
+                if (value.Length > MaximumRequestQueueNameLength)
```

Comment:
Confused why were are protecting against this. What happens if someone sets a request queue larger than 260 characters?

Reply:
> What happens if someone sets a request queue larger than 260 characters?

We know it'll fail (Windows MAX_PATH). This is just optimistically protecting against it.

Reply:
What's the native error I'd get if I tried?

Reply:
`ErrorCodes.ERROR_INVALID_PARAMETER`

Reply:
Can someone change the max path for that parameter via a regkey? If so, I wouldn't throw here and just let HttpSys throw.

Reply:
Fair enough. I'll remove this check